tox.ini

```diff
@@ -76,7 +76,7 @@ deps =
     behave==1.2.4
     behave-pytest==0.1.1
 commands =
-    python -m behave {posargs}
+    behave {posargs}
```

Comment:
my bad, I was testing some itest changes earlier, and changed it just to see if it works like this. Will remove before committing.